_posts/2020-01-01-2019-summary.md

```diff
@@ -0,0 +1,178 @@
+[San Francisco]({% post_url 2019-05-17-Presto-Summit %}), [Tokyo]({% post_url 2019-07-11-report-for-presto-conference-tokyo %}), 
+[Bangalore]({% post_url 2019-09-05-Presto-Summit-Bangalore %}) and New York) which 
+brought together thousands of members from the larger Presto community.
+
+The project has been more active than ever. We completed 28 releases comprised of more than 2850 
+commits in over 1500 pull requests. Of course, that alone is not a good measure of progress, so 
+let’s take a closer look at everything that went in.
+
+<!--more-->
+
+# Improvements
+
+## Language Features
+
+* [`FETCH FIRST n ROWS [ONLY] [WITH TIES]`](/docs/current/sql/select.html#limit-or-fetch-first-clauses) 
+standard syntax. The `WITH TIES` clause is particularly useful when some of the rows have the same 
+value for the columns being used to order the results of a query.
+	(TODO: sample query)
+* [`OFFSET`](/docs/current/sql/select.html#offset-clause) syntax.
+* [`COMMENT ON <table>`](/docs/current/sql/comment.html) syntax to 
+  set or remove table comments, and ability to see table comments via `DESCRIBE` or the 
+  `system.metadata.table_comments` table.
+* Support for ``LATERAL`` in the context of an outer join.
+* Support for ``UNNEST`` in the context of ``LEFT JOIN``. With this feature, it is now possible 
+  to preserve the outer row when the array contains zero elements or is NULL.
+	
+  ```sql	
+  SELECT * FROM t LEFT JOIN UNNEST(t.a) u(v) ON true
+  ```
+
+* Window functions now support the IGNORE NULLS clause. This is useful when combined with 
+  functions such as LEAD, LAG, FIRST_VALUE, LAST_VALUE and NTH_VALUE if the dataset contains nulls.
+* `.*` for ROW expressions.
+* `ROW` types now support the subscript operator to access fields by index. This improves usability 
+  when dealing with ROW type containing anonymous fields.
+  
+    ![](/assets/blog/2019-review/row-ordinal.png)
+
+* [`CREATE SCHEMA`](/docs/current/sql/create-schema.html) syntax and support 
+  in various connectors (Hive, Iceberg, MySQL, PostgreSQL, Redshift, SQL Server, Phoenix).
+
+## Query Engine
+
+* Extend support for conditional, lazy loading and processing (a.k.a., late materialization) to 
+  Join, Window, TopN and Semijoin operators. This can dramatically reduce latency, CPU and I/O 
+  for highly selective queries.
+* [Unwrap cast/predicate pushdown]({% post_url 2019-05-21-optimizing-the-casts-away %}) optimizations.
+* Limit, table sample, projection pushdown into connectors.
+* [Dynamic filtering]({% post_url 2019-06-30-dynamic-filtering %}) for broadcast joins.
+
+## Performance
+
+* Improve performance of [`UNNEST`]({% post_url 2019-08-23-unnest-operator-performance-enhancements %}) by 10x.
+* Improve performance of [ORC decoders]({% post_url 2019-04-23-even-faster-orc %}) by 2-7x resulting on a 
+  10% global CPU improvement for the TPC-DS benchmark.
+* Improvements when reading small row groups in Parquet (e.g. small Parquet files, files with 
```

Comment:
The braced expl is not necessary, but let's put this in context:

```suggestion
* Up to 17x speedup when reading small Parquet files or files with large number of columns, especially visible on S3 or other object storage.
```

17x -- this is a number from real production use-cases, i can describe this more.